central/auth/m2m/claim_extractor_test.go

```diff
@@ -0,0 +1,73 @@
+package m2m
+
+import (
+	"testing"
+
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/errox"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_newClaimsExtractorFromConfig(t *testing.T) {
```

Comment:
nit: naming alignment
```suggestion
func Test_newClaimExtractorFromConfig(t *testing.T) {
```